### ISSUE TITLE
Some CCGs missing on choropeth maps

### ISSUE THREAD

**Issue by richiecroker** [ebmdatalab/datalab-pandas#6]
Some CCGs appear to be missing on the choropeth function:
![image](https://user-images.githubusercontent.com/18683239/53747486-46314d80-3e9b-11e9-96a2-b06a1d037569.png)

I'm assuming this is due to mergers since the file was originally created - i think one missing one is Buckinghamshire CCG, which was formed in April 2018.

**Comment by HelenCEBM** [comment]
yep newly-formed berkshire east ccg and berkshire west ccg also missing

**Comment by brianmackenna** [comment]
Major CCG changes occurred in April 2020. This is the [first notebook](https://github.com/ebmdatalab/oramorph-notebook/blob/master/notebooks/Oramorph.ipynb) the error has reoccured.
